frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/index.tsx

```diff
@@ -31,6 +33,18 @@ export const FlashSystemConnectionDetails: React.FC<ExternalComponentProps<Flash
       : setEndpointValid(ValidatedOptions.error);
   };
 
+  const volModeList = ['thick', 'thin'];
```

Comment:
```suggestion
  const  VOLUME_MODES = ['thick', 'thin'];
```
Define it outside the component